crates/teloxide-macros/src/bot_commands.rs

```diff
@@ -50,9 +50,9 @@ fn impl_commands(infos: &[Command]) -> proc_macro2::TokenStream {
     });
 
     quote! {
-        fn bot_commands() -> Vec<teloxide::types::BotCommand> {
+        fn bot_commands() -> ::std::vec::Vec<teloxide::types::BotCommand> {
```

Comment:
I'd like to propose the approach I do when writing proc macros.
I define such a "doc-hidden-public" module in the main crate:
```rs
#[doc(hidden)]
pub mod imp {
     pub use std;
}
```

Then use the pattern of creating a `p` variable referring to the standard prelude module and using it to shorten the paths:
```
let p = quote!(::main_crate_name::imp::std::prelude::rust_2021)

quote! {
    fn foo() -> #p::Option {
         #p::None
    }
}
```

Reply:
Interesting trick, but what’s appends if the final user of your crate use another rust edition?

Reply:
The same `::teloxide::imp::std::prelude::rust_2021` path will be pasted in the generated code regardless of the edition of the crate where this proc macro was invoked. However, it will still work if the edition of the user crate is 2015 or 2018, because the module `std::prelude::rust_2021` is still available in older editions too.

If you wonder what the final generated code will look like. It'll be this (according to the example in my comment):
```rs
quote! {
    fn foo() -> ::teloxide::imp::std::prelude::rust_2021::Option {
       ::teloxide::imp::std::prelude::rust_2021::None
    }
}
```
And this code does compile on any edition ✔️

Reply:
An interesting trick! I won't block this PR on it, but feel free to open another one later.